$OpenBSD: patch-ext_nokogiri_extconf_rb,v 1.1 2014/11/02 00:05:49 jeremy Exp $

Remove writing to /dev/tty during build, it screws up dpb display.

--- ext/nokogiri/extconf.rb.orig	Sat Nov  1 17:01:10 2014
+++ ext/nokogiri/extconf.rb	Sat Nov  1 17:01:50 2014
@@ -41,13 +41,6 @@ HELP
 end
 
 def message!(important_message)
-  message important_message
-  if !$stdout.tty? && File.chardev?('/dev/tty')
-    File.open('/dev/tty', 'w') { |tty|
-      tty.print important_message
-    }
-  end
-rescue Errno::ENXIO
 end
 
 def do_clean
